src/FastCache.spec.ts

```diff
@@ -241,4 +241,33 @@ describe('FastCache', () => {
       }
     });
   });
+
+  describe('flush', () => {
+    test('limit bigger than 50', (done) => {
```

Comment:
테스트 함수 이름 조금만 더 명확하게 작성해주시면 더 좋을것 같습니다!
(한글도 좋구요!)
```suggestion
    test('flush would be called when limit bigger than 50', (done) => {
```

Reply:
퍼블릭 레포는 영어로 한다는 강요없는 컨벤션에 따라 영어로 상세하게 작성했습니다 😄